portfolio/src/main/webapp/cook.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Food Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+ <body>
+
+ <div class="foodcontent" >
+     <br/>
+    <h2 class="center">Food Gallery</h2>
+    <img class="pfp center" style="width:250px;height:250px;" src="img/shef.png" />
+    <h3 class="center">Some food I've made over quarantine ... </h3>
+
+    <div id="imggrid">
+        <img class="foodpic" src="img/rice.jpg"/>
```

Comment:
Consider trying to automate some of these repeated <img> tags with some javascript. You could create a list with all the image srcs and then append them programmatically to the imiggrid div

Reply:
I tried automating it w/ JS but the imggrid layout is not retained. This is probably best for a straight layout.

Reply:
Thats ok, you would probably have more luck if we were using a frontend framework anyway. This is fine for now.